unit_tests/test_ssh.py

```diff
@@ -0,0 +1,66 @@
+import unittest
+from pathlib import Path
+import shutil
+import stat
+import tempfile
+
+from archive_auth_mirror.ssh import (
+    add_authorized_key,
+    create_key,
+)
+
+
+class CreateKeyTest(unittest.TestCase):
+
+    def setUp(self):
+        self.tempdir = Path(tempfile.mkdtemp())
+        self.addCleanup(shutil.rmtree, str(self.tempdir))
```

Comment:
that's ok, but you should use TempDir() then (there are other tests that do that)